src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -237,7 +237,7 @@ void drawing_classifier::init_options(
   options.create_string_option(
       "warm_start",
       "Record warm start model version used. If no warmstart used"
-      " None is assigned by default."
+      "Empty string is assigned by default.",
       "",
```

Comment:
The default value is an empty string. I assume empty string will be used instead of UNDEFINED or "None".

Reply:
@fareeha @nickjong @Jarvi-Izana 

Are we changing the behavior of the toolkit to use "" (which I assume later translates to FLEX_UNDEFINED, and mimics what used to be `warm_start=None` behavior) as the default value instead of "auto"?

Reply:
like python, the empty string is not None. The default value is passed from the python layer. It should be fine.

Reply:
I think I original suggested that the empty string be used here, but in hindsight, I wonder if `FLEX_UNDEFINED` works here?

Reply:
It should work. Which way do we want to go?